DEPENDENCIES.rst

```diff
@@ -106,6 +114,14 @@ and (optionally):
 #. libblas-dev
 #. liblapack-dev
 #. libgoogle-perftools-dev
+#. python3
+#. python3-dev
+#. python3-tables
+#. python3-pandas
+#. python3-numpy
+#. python3-nose
+#. python3-jinja2
+#. cython3
 
```

Comment:
I guess the default version depends on whether you are on ubuntu server or not: https://wiki.ubuntu.com/XenialXerus/ReleaseNotes

Removing the 3 will go to the Python 2 version, but on some level it is up to the user to know whether they want the binding to 2 or 3. If you are using the 3 interpreter, the 2 packages won't help. Maybe they could be listed as "python3-numpy or python-numpy"